### PR TITLE
Remove rtd build test on osx

### PR DESCRIPTION
**Pull request by okuta**
This PR reduces test time.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -63,8 +63,10 @@ script:
   - PYTHONWARNINGS='module::DeprecationWarning' nosetests -a '!gpu,!slow' --with-doctest tests/install_tests
   - cd tests
   - PYTHONWARNINGS='module::DeprecationWarning' nosetests -a '!gpu,!slow' --with-doctest chainer_tests
-  - cd ..
-  - READTHEDOCS=True python setup.py develop
+  - if [[ $TRAVIS_OS_NAME == "linux" ]]; then
```
Comment (unnonouno) [inline .travis.yml]: Can you insert echo here? I cannot confirm that this line is really executed.

Comment (okuta) [inline .travis.yml]: If this line was executed, some status was outputted.
Please see https://travis-ci.org/pfnet/chainer/jobs/213242912#L877
Is it not enough?

Comment (unnonouno) [inline .travis.yml]: OK

**Comment by unnonouno** [comment]
LGTM

**Pull request merged**